code-gurus/artlane_backend
# add continuous integration

# PR Details

This PR consists of setting up the testing environment and continuous integration for our platform.

## Related Issue or Trello raises! 

For better control and good codebase, we need continuous integration for good practice and to control any failing point that can occur while building our app.


## Motivation and Context | Why are these changes important?

We need to increase the quality of our code.

## How can this be tested locally?

We are using Travis CI. So if you go on it and run the build of the PR it should pass.

## Types of changes


## Checklist





## Screenshots

N/A
